sources/packages/backend/apps/db-migrations/src/sql/SFASIndividuals/Update-swap-csl-bcsl-columns.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE
+    sims.sfas_individuals
+SET
+    csl_overaward = bcsl_overaward,
+    bcsl_overaward = csl_overaward;
```

Comment:
Sonar is complaining about the "where" clause. Maybe adding: "WHERE 1=1" would solve it.

Reply:
Please do not change the code. I resolved the sonar cloud issue. In this case, it is a false positive since we want the query to be executed without a where clause.